docs/getting_started/install_on_ibexa_cloud.md

```diff
@@ -0,0 +1,74 @@
+# Install on Ibexa Cloud
+
+Ibexa Cloud enables you to host your application in the cloud using the [Platform.sh](https://platform.sh/) service.
+
+## 1. Prepare configuration files
+
+If you hadn't run the `composer ibexa:setup` command during installation, run it now:
+
+``` bash
+composer ibexa:setup --platformsh
+```
+
+This command adds to your project configuration files required for using Ibexa Cloud.
+
+You can adapt the configuration in two places:
+
+- `.platform.app.yaml` - main configuration
+- `.platform/services.yml` - additional services such as search engines or cache
```

Comment:
There is also `.platform/routes.yaml`, not sure if it's worth mentioning here (it should be modified when enabling Fastly, for example).